params/governance_params.go

```diff
@@ -112,61 +106,6 @@ var (
 	DefaultDeriveShaImpl             = uint64(0)     // Orig
 )
 
-func IsStakingUpdateInterval(blockNum uint64) bool {
```

Comment:
For tidyng up instead of inlining, is it not better refactor and revive this function by adding additional param for interval value?